sdk/src/main/java/com/mapbox/maps/renderer/widget/BitmapWidgetRenderer.kt

```diff
@@ -80,12 +80,20 @@ internal class BitmapWidgetRenderer(
 
   private fun updateVertexBuffer() {
     // in pixels, (-bitmapWidth / 2, -bitmapHeight/2) - (bitmapWidth / 2, bitmapHeight/2)
-    vertexPositionBuffer.put(
-      -bitmapWidth / 2f, -bitmapHeight / 2f,
-      -bitmapWidth / 2f, bitmapHeight / 2f,
-      bitmapWidth / 2f, -bitmapHeight / 2f,
-      bitmapWidth / 2f, bitmapHeight / 2f,
-    )
+    when (position.horizontal) {
+      WidgetPosition.Horizontal.CENTER -> vertexPositionBuffer.put(
+        0f, 0f,
+        0f, bitmapHeight.toFloat(),
+        bitmapWidth.toFloat(), 0f,
+        bitmapWidth.toFloat(), bitmapHeight.toFloat()
+      )
+      else -> vertexPositionBuffer.put(
+        -bitmapWidth / 2f, -bitmapHeight / 2f,
+        -bitmapWidth / 2f, bitmapHeight / 2f,
+        bitmapWidth / 2f, -bitmapHeight / 2f,
+        bitmapWidth / 2f, bitmapHeight / 2f
+      )
+    }
```

Comment:
can we cover it with some tests?

Reply:
Agreed, but can we do that with Android Auto?

Reply:
I don't think you need Android Auto really. You could test directly with `MapSurface`. Something like this:
```Kotlin
  @Test
  fun mapSurfaceBitmapWidget() {
    val latch = CountDownLatch(1)
    rule.scenario.onActivity { activity ->
      activity.runOnUiThread {
        val surfaceView = SurfaceView(activity)
        lateinit var mapSurface: MapSurface
        surfaceView.holder.addCallback(object : SurfaceHolder.Callback {
          override fun surfaceCreated(holder: SurfaceHolder) {
            val options = MapInitOptions(context = activity, styleUri = null)
            mapSurface = MapSurface(activity, holder.surface, options)
            mapSurface.onStart()
            mapSurface.surfaceCreated()
            mapSurface.addWidget(
              BitmapWidget(
                createBitmap(),
                position = WidgetPosition(
                  WidgetPosition.Horizontal.CENTER,
                  WidgetPosition.Vertical.CENTER
                )
              )
            )
          }

          override fun surfaceChanged(holder: SurfaceHolder, format: Int, width: Int, height: Int) {
            mapSurface.surfaceChanged(width, height)
          }

          override fun surfaceDestroyed(holder: SurfaceHolder) {
            mapSurface.surfaceDestroyed()
          }

        })
        activity.frameLayout.addView(surfaceView)
      }
    }
    if (!latch.await(LATCH_TIMEOUT_SEC, TimeUnit.SECONDS)) {
      throw TimeoutException()
    }
  }
```

Reply:
True, good point @jush , will try that out!

Reply:
Note that both MapView and MapSurface support adding Widgets